### ISSUE TITLE
Fix or Remove Default Simulation Params

### ISSUE THREAD

**Issue by nagakingg** [curveresearch/curvesim#222]
### Version Information

curvesim 0.5.0.b1; all OSs and Python versions 

### What's your issue about?

This bug was user-reported, and I verified it.

Running a v2 autosim with no parameter input raises a ParameterSamplerError:
```python
res = curvesim.autosim(
    "0x5426178799ee0a0181a89b4f57efddfab49941ec",
    env="staging",
)

curvesim/templates/param_samplers.py", line 129, in _validate_attributes
    raise ParameterSamplerError(
curvesim.exceptions.ParameterSamplerError: Input parameters not found in 'ParameterizedCurveCryptoPoolIterator.setters' or 'SimCurveCryptoPool' attributes: ['fee']


```

### How can it be fixed?

This is caused by the v1 parameter defaults being applied to the v2 pool in both pipelines `pipeline` functions:

```python
    default_params = DEFAULT_PARAMS.copy()
    for key in DEFAULT_PARAMS:
        if key in fixed_params:
            del default_params[key]

    variable_params = variable_params or DEFAULT_PARAMS
```
https://github.com/curveresearch/curvesim/blob/1bddb535b692be8d56850abf7b40e4c4b4f9ebb6/curvesim/pipelines/vol_limited_arb/__init__.py#L118
https://github.com/curveresearch/curvesim/blob/1bddb535b692be8d56850abf7b40e4c4b4f9ebb6/curvesim/pipelines/simple/__init__.py#L92

There are two options:
- make default parameters for each pool type
- remove the default parameters altogether

My sense is that we should simply remove the default parameters (and the above code block that applies them) since they are rarely/never used. This would mean that running a pipeline with no parameter kwargs would run one run with the pool's fetched parameters. This would require a minor update to the readme/documentation.

In the future we might include some standard settings to choose from, but I think they could be defined in a separate submodule and shouldn't be applied by default.

**Comment by chanhosuh** [comment]
Let's remove `autosim` and also the test option.  That can be explicitly chosen and passed in for tests.

**Comment by nagakingg** [comment]
Noting the results of our discussion:

**Immediate change:** remove default and test parameters. This issue can be closed with this completed.

Subsequent changes:
- remove autosim (#225)
- remove fixed/variable parameters distinction (#226)
